app.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "Showdown bot",
+  "description": "A Pokemon Showdown Battle Bot written in Python",
+  "repository": "https://github.com/pmariglia/showdown",
+  "env": {
+    "WEBSOCKET_URI": {
+      "description": "The address to use to connect to the Pokemon Showdown websocket",
+      "value": "sim.smogon.com:8080",
```

looks like a typo - the PokemonShowdown websocket server runs on port 8000
```suggestion
      "value": "sim.smogon.com:8000",
```